get-started/part2.md

```diff
@@ -318,6 +318,10 @@ If the image isn't available locally on the machine, Docker will pull it from
 the repository.
 
 ```shell
+docker image rm <image id>
```

Comment:
I'm not sure it's worth redoing the video. There are only 2 commands that could be updated: (1) docker ps [1:50] and docker stop <hash> [1:56].